accounting/defines.py

```diff
@@ -0,0 +1,31 @@
+DOCUMENT_TYPE_CHOICES = (
```

Comment:
Es recomendable para este tipo de atributos dejarlos como lista de tupla
```suggestion
DOCUMENT_TYPE_CHOICES = [
(),
(),
()... 
]
```